conu/backend/podman/image.py

```diff
@@ -410,3 +406,19 @@ def layers(self, rev=True):
         if not rev:
             image_layers.reverse()
         return image_layers
+
+    @property
+    def metadata(self):
+        if self._metadata is None:
+            self._metadata = ImageMetadata()
+            self._metadata = self.get_metadata()
+        return self._metadata
+
+    def get_metadata(self):
+        """
+        Provide metadata about this image.
+
+        :return: ImageMetadata, Image metadata instance
+        """
+        inspect_to_metadata(self._metadata, self.inspect(refresh=True))
```

Comment:
this is gonna blow when self._metadata is None, please change this to
```
if self._metadata is None:
  self._metadata = ImageMetadata()
inspect_to_metadata(self._metadata, self.inspect(refresh=True))
```

and remove line `self._metadata = ImageMetadata()` from the prop above